test_runner/fixtures/neon_fixtures.py

```diff
@@ -1839,6 +1937,9 @@ def __init__(self, env: NeonEnv, port: PageserverPort, config_override: Optional
             ".*compaction_loop.*Compaction failed, retrying in.*timeline is Stopping",  # When compaction checks timeline state after acquiring layer_removal_cs
             ".*query handler for 'pagestream.*failed: Timeline .* was not found",  # postgres reconnects while timeline_delete doesn't hold the tenant's timelines.lock()
             ".*query handler for 'pagestream.*failed: Timeline .* is not active",  # timeline delete in progress
+            # FIXME: on-demand download causes these, need to triage
+            ".*could not compact, repartitioning keyspace failed: layer file needs to be downloaded",
+            ".*Gc failed, retrying in .*: layer file needs to be downloaded",
```

Comment:
Do these log messages still apply?
@SomeoneToIgnore changed the log message text for `NeedsDownload` recently.

Can't find them in the codebase.

Remove these from `allowed_errors`

Reply:
Removed them.